### ISSUE TITLE
Staying up to date

### ISSUE THREAD

**Issue by timoxley** [solutionio/async#2]
I'd like to use this component, but I don't have confidence the repo will be kept in sync with the official caolan/async… IMO having two independent repos is the worse than the problem components is trying to solve. It'd be better if this was simply a branch of the official async so anyone can easily PR updates. have you mentioned this to @caolan?

**Comment by sebs** [comment]
There Re several issues in tis. 
1. A clone.
   My Bad, i was just hacking together code for a own project at hat time. Surely I can clone the repo and go on from there.
2. Compat.
   he code of the original lib needs a change, as long as this is the case there is not much the lriginal author can do. Its really ugly to pollute theoriginal, veryclean lib with he module.expors statement. I doubt it will find its way in the original codenase
3. Mentioning
   I will mentoon it, after a clone to the original author and we will see whats happening.

**Comment by timoxley** [comment]
> Its really ugly to pollute theoriginal, veryclean lib with he module.expors statement

Agreed, a new branch in the original repo that contains the `component` compatible changes would fix this.

> Surely I can clone the repo and go on from there.

Surely you can, but divergent codebases are bad since bug fixes and api changes won't necessarily align.
